src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Todo {
+    private String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+    public char getTaskType() {
+        return 'D';
```

Comment:
Consider returning a character set to 'D' instead of just putting it in the return statement